src/System.IO.Pipelines/src/System/IO/Pipelines/ThreadPoolScheduler.netcoreapp.cs

```diff
@@ -13,5 +13,10 @@ public override void Schedule(Action<object> action, object state)
         {
             System.Threading.ThreadPool.QueueUserWorkItem(action, state, preferLocal: false);
         }
+
+        internal override void UnsafeSchedule(Action<object> action, object state)
+        {
+            System.Threading.ThreadPool.UnsafeQueueUserWorkItem(action, state, preferLocal: false);
+        }
```

Comment:
Should the inline scheduler override to just invoke the delegate without doing all the flow suppression stuff?

Reply:
Wasn't sure on that?